app/src/main/java/com/example/therundown/domain/NbaAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.therundown.domain
```

Comment:
this class should be in the view layer
because it contains UI logic

Reply:
done